### PR TITLE
Remove testnet question

### PR DESCRIPTION
**Pull request by yahiheb**
Most of what is said there is already in this chapter
https://docs.wasabiwallet.io/using-wasabi/Testnet.html

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -1255,25 +1255,6 @@ You can use the [Wasabi RPC server `gethistory` call](/using-wasabi/RPC.md#gethi
 
 ## Settings
```
Comment (MaxHillebrand) [inline docs/FAQ/FAQ-UseWasabi.md:1257]:
```suggestion

:::details
### How to switch to Testnet?

You can toggle between the Bitcoin `mainnet, testnet, or local regtest in the `Settings` tab.
This changes apply after a restart of Wasabi.

![](/SettingsNetwork.png)

Read more [here](/using-wasabi/Testnet.md).
:::
```

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md:1257]: IMO this is not necessary.

Comment (MaxHillebrand) [reply docs/FAQ/FAQ-UseWasabi.md:1257]: We also have something similar to the full node. 
A quick guide on the ways to activate it, and a link to the standalone chapter.
Especially in light of the search function, I think this is good to have.

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md:1257]: I checked the full node FAQs and they add some value IMO, but for the testnet FAQ there is nothing that is not in the standalone chapter.

**Pull request merged**